### PR TITLE
[freeimage/libraw] Fix libraw capitalization in find_package calls, don't provide FindLibRaw.cmake

### PR DESCRIPTION
**Pull request by russelltg**
- **[freeimage] use the correct capitalization of libraw**
- **./vcpkg x-add-version --all**

It's supposed to be `libraw`, but ended up being `LibRaw` in all the find_package calls

- [x] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md).
- [x] SHA512s are updated for each updated download.
- [x] The "supports" clause reflects platforms that may be fixed by this new version.
- [x] Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.
- [x] Any patches that are no longer applied are deleted from the port's directory.
- [x] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [x] Only one version is added to each modified port's versions file.

### CODE REVIEW

**Comment by russelltg** [comment]
It seems the CONFIG version of libraw doesn't set LibRaw_LIBRARIES, but exposes the target instead. 

I will open a separate PR for removing installing the FindLibRaw.cmake, which doesn't really make sense to be packaged with libraw....

**Comment by dg0yt** [comment]
> I will open a separate PR for removing installing the FindLibRaw.cmake, which doesn't really make sense to be packaged with libraw....

I guess the libraw keeps it for compatibility. It is usable via the vcpkg cmake wrapper.

File: ports/freeimage/CMakeLists.txt [hunk 1]
```diff
@@ -16,7 +16,7 @@ find_package(TIFF           REQUIRED)
 find_package(OpenJPEG       REQUIRED)
 find_package(WebP CONFIG    REQUIRED)
 find_package(JXR            REQUIRED)
-find_package(LibRaw         REQUIRED)
+find_package(libraw         REQUIRED)
```
[the remark this answers is not in the document]
Comment (dg0yt) [reply ports/freeimage/CMakeLists.txt]: Note that `NAMES` also activates CONFIG mode.

**Comment by russelltg** [comment]
My understanding is `Find*` scripts are not meant to be packaged with the library itself? Aren't those typically for consuming packages without native cmake find scripts?

**Comment by dg0yt** [comment]
> My understanding is `Find*` scripts are not meant to be packaged with the library itself? Aren't those typically for consuming packages without native cmake find scripts?

This might be mixing different aspects.
- Modern Config mode offers more search options than traditional Find modules.
- Modern Config is normally exported and installed by the dependency. Find modules are normally meant to be copied into user projects.
- Users who rely on the Find module integration via the wrapper get locked into vcpkg. This is not desired.
- Removing the Find moduleintegration via the wrapper will break users which rely on this convenience.

If there is a sane alternative, I announce it via `usage`, and sometimes I add a deprecation warning into the legacy cmake code path.

**Comment by LilyWangLL** [comment]
Can you please resolve the conflicts against master?

**Comment by LilyWangLL** [comment]
Pinging @russelltg for response. Is work still being done for this PR?

**Comment by LilyWangLL** [comment]
Closing this PR since it seems that no progress is being made. Please ping us to reopen if work is still being done.

**Pull request closed without merging**